### ISSUE TITLE
Watch for fuel tanks and other industrial infrastructure

### ISSUE THREAD

**Issue by almccon** [osmlab/labuildings#77]
Something to look for when we import and validate. Some things like this object https://www.openstreetmap.org/way/408511408 are tagged with `building=industrial`, and they should probably have the tag `man_made=storage_tank` as well.

See http://wiki.openstreetmap.org/wiki/Tag:man_made%3Dstorage_tank, which suggests that it's acceptable to leave the `building` tag, too.

@imagico spotted this [on the mailing list](https://lists.openstreetmap.org/pipermail/imports/2016-April/004424.html). 

![screen shot 2016-04-12 at 12 apr 4 27 33](https://cloud.githubusercontent.com/assets/1212178/14478568/d11e843c-00cb-11e6-9ddb-c65eafdcd3b8.png)

**Comment by talllguy** [comment]
Good catch. Any way to check automatically for round features and flag them for tagging as tanks, silos or smokestacks?

**Comment by almccon** [comment]
Maybe catching round things could be done after the fact using Mapbox QA tiles?

**Comment by maning** [comment]
We can definitely catch round buildings, but automatic tagging maybe risky.  What we can do is find all these round buildings and have a micro-task to do manual review by mappers.

**Comment by imagico** [comment]
My idea was mainly to supplement these with a fixme tag to indicate the default tags are likely insufficient.  Not every round structure in an industrial area is a storage tank.

The industrial areas are fairly concentrated so it might already be sufficient if those who take on these parts plan enough time to work this out by hand - ideally do some research in what kind of product/process is done there and tag accordingly (i.e. also content=* etc.)  In general with just buildings mapping of industrial facilities will never achieve any reasonable level of completeness in comparison to residential or commercial areas - there are so many different mappable structures there and buildings are just one of them.

**Comment by maning** [comment]
@imagico, I like adding `fixme` for `building=industrial`.  I can run that for the rest of the imported data.  For the already imported data, we can ask validators to work on this specific task.

**Comment by bdiscoe** [comment]
FWIW, last week, I did a bunch of the import tiles around the refineries (lots of tanks), and I manually changed building=industrial to man_made=storage_tank - at least, I did it for most of them.  It's a quick and easy thing to do.
